src/style/app.module.css

```diff
@@ -31,6 +31,7 @@
   --tablerow-bg-color: #eff1f7;
   --row-background: var(--bs-white, white);
   --font-size-header: 16px;
+  --primary-border-green: 1px solid #31bb6b;
```

Comment:
1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466 
1. This needs to be changed to `--primary-border` or something non color specific or use an existing border color. We are moving away from green. There are other border colors

Reply:
Okay sure I'll look into that